install.sh

```diff
@@ -558,9 +558,13 @@ do_install() {
 				exit 1
 			fi
 
-			sles_version="${dist_version##*.}"
 			sles_repo="$DOWNLOAD_URL/linux/$lsb_dist/$REPO_FILE"
-			opensuse_repo="https://download.opensuse.org/repositories/security:SELinux/SLE_15_SP$sles_version/security:SELinux.repo"
+			if [ "$dist_version" = "15.3" ]; then
+				sles_version="SLE_15_SP3"
+			else
+				sles_version="15.4"
+			fi
+			opensuse_repo="https://download.opensuse.org/repositories/security:SELinux/$sles_version/security:SELinux.repo"
```

Comment:
Does it need to be prepared for other version, or do you prefer to keep this strict?

Perhaps we could change this to;

```suggestion
			case "$dist_version" in
				"15.3")
					opensuse_repo="https://download.opensuse.org/repositories/security:SELinux/SLE_15_SP3/security:SELinux.repo"
					;;
				"15.4")
					opensuse_repo="https://download.opensuse.org/repositories/security:SELinux/15.4/security:SELinux.repo"
					;;
				*)
					echo "This script only supports SLES 15 SP3 and SP4"
					exit 1
					;;
			esac
```

Reply:
For SLES-15SP4, '15.4' is considered in the repo URL. If we want to be prepared for further releases, we could do something as below:

```
if [ "$dist_version" = "15.3" ]; then
	sles_version="SLE_15_SP3"
else
        sles_minor_version="${dist_version##*.}"
	sles_version="15.$sles_minor_version"
fi
opensuse_repo="https://download.opensuse.org/repositories/security:SELinux/$sles_version/security:SELinux.repo"
```

Note: SLES 15-SP3 will be EOL on 2022/12/31. We can revert the strict check of 15.3 or 15.4 to make it generic for further releases.

Reply:
Ah, yes; I'm fine either way, if you're comfortable that "SLES 15 SP5" will work with the existing packages, then your suggestion looks good to me.

I recall there could sometimes be quite some changes between their "SP"'s, not always making them "minor" updates, which is why I was wondering if you preferred to be "strict" (and print an error message for other versions)

@Prabhav-Thali feel free to update your PR with one of those options 👍

Reply:
Updated the PR